src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -82,6 +82,7 @@ $overlayShapeZIndex: 1;
 .HeroRecommendation-heading .AddonTitle {
```

Comment:
I see that you were fighting some CSS specificity issues here. In the future, I'd suggest passing a custom class to `AddonTitle` which would decouple `HeroRecommendation` from the implementation of `AddonTitle`. You could do something like `<AddonTitle className='HeroRecommendation-addonTitle' .../>` then declare it here:

```css
.HeroRecommendation-heading,
.HeroRecommendation-addonTitle {
  // ...
}
```

If you still need more specificity just start grabbing parent selectors:

```css
.HeroRecommendation {
  .HeroRecommendation-heading,
  .HeroRecommendation-addonTitle {
    // ...
  }
}
```